packages/sdk/package.json

```diff
@@ -37,8 +22,6 @@
     "test": "jest --forceExit",
     "test:ci": "CI=1 jest --forceExit",
     "build": "rm -rf dist && tsup",
-    "start": "node ./dist/cjs/swap/index.js",
```

Comment:
these did not do anything, so i think we can remove them and their associated packages